include/state-observation/observer/kalman-filter-base.hpp

```diff
@@ -309,13 +309,17 @@ class STATE_OBSERVATION_DLLAPI KalmanFilterBase : public ZeroDelayObserver, prot
   /// get the last predicted measurement
   MeasureVector getLastPredictedMeasurement() const;
 
+  MeasureVector getLastMeasurement() const;
+
   /// get the last Kalman gain matrix
   Matrix getLastGain() const;
 
   /// set update functions for sum and difference for the state vector
   /// (used for the case of multiplicative Kalman filter)
   void setStateArithmetics(StateVectorArithmetics * arith);
 
+  virtual void resetPrediction();
```

Comment:
+ Missing documentation.

I am not sure what this function is doing, but since this is a base function maybe it needs to be virtual in case there are additional things to do when resetting the prediction

Reply:
Sure but `resetPrediction()` is not used inside `state-observation` so it seems like an odd choice